tools/ci_build/github/azure-pipelines/linux-cpu-lort-pipeline.yml

```diff
@@ -0,0 +1,64 @@
+jobs:
```

Comment:
You can add it to orttraining-linux-ci-pipeline.yml

Reply:
What do you mean? I have a separated CI for this feature, just like aten-fallback and eager CIs. Unless it's stable enough, I prefer to keep it away from other pieces.

Reply:
Why training needs to have so many pipelines?

Reply:
This pipeline is completely different. No other training/inference pipeline requires the latest Pytorch commit. I am trying to merge it with eager_mode pipeline. Hopefully it can resolve your concern.